security/pkg/nodeagent/cache/secretcache_test.go

```diff
@@ -189,6 +190,19 @@ func TestWorkloadAgentRefreshSecret(t *testing.T) {
 	if retries == 5 {
 		t.Errorf("Cached secret failed to get refreshed, %d", atomic.LoadUint64(&sc.secretChangedCount))
 	}
+
+	key := ConnKey{
+		ProxyID:      testProxyID,
+		ResourceName: testResourceName,
+	}
+	if _, ok := sc.secrets.Load(key); ok != true {
+		t.Errorf("Failed to find secret for %+v from cache", key)
+	}
+
+	sc.DeleteSecret(testProxyID, testResourceName)
+	if _, ok := sc.secrets.Load(key); ok != false {
```

Comment:
`ok != false` -> `ok` ? Or, rename the variable to `found`?

Reply:
renamed